update Croatian holidays for 2020.

gour opened the pull request:
Hello!

Here is the update for Croatian holidays for the year 2020...hoping that website will generate it and make it available for usage.


Sincerely,
Gour

lauft commented on python/holidays/hr-HR.py:
```diff
@@ -9,12 +9,13 @@ class hr_HR(Locale):
     01-01: [NF] Nova Godina
     01-06: [NRF] Bogojavljanje / Sveta tri kralja
     05-01: [NF] Praznik rada
+    05-30: [NF] Dan državnosti   
```
Is the date of this holiday wrong or does it change from year to year (i.e. is not fixed)? If it does change: what is the algorithm to determine its date?

lauft commented on python/holidays/hr-HR.py:
```diff
@@ -9,12 +9,13 @@ class hr_HR(Locale):
     08-15: [NRF] Velika Gospa
     10-08: [NF] Dan neovisnosti
     11-01: [NRF] Dan svih svetih
+    11-18: [NF] Dan sjećanja na žrtve Domovinskog rata i Dan sjećanja na žrtvu Vukovara i Škabrnje
```
Is this a new holiday for 2020 or did it exist before? Is this the official name (it seems unusual long...)?

gour replied:
> Is this a new holiday for 2020 or did it exist before? Is this the official name (it seems unusual long...)?
Yes, this is a new on valid from 2020 onwards and such is the name - you can check the above-mentioned official document.

Sincerely,
Gour

gour commented:
>     * Do they affect 2020 only, all years from 2020 on, or also the previous years?

Well, the Croatian government changed the relevant law in Nov 2019., creating a new (fixed) holiday on Nov 18th and removed some old one.

>     * Maybe you could provide the legal sources which define the holidays?

Here is the official [law](https://www.zakon.hr/z/372/Zakon-o-blagdanima,-spomendanima-i-neradanim-danima-u-Republici-Hrvatskoj) document.

> PS: We have just now converted holidata into a python-only project. You have to "move" your changes from ` python/holidays/hr-HR.py` to `holidata/holidays/hr-HR.py` due to the restructuring.

OK.
